commons/all/src/main/java/org/infinispan/commons/configuration/io/ConfigurationReader.java

```diff
@@ -179,11 +197,30 @@ default boolean inTag(Enum<?> name) {
    void handleAttribute(ConfigurationReaderContext context, int i);
 
    default void require(ElementType type) {
-      require(type, null, null);
+      require(type, null, (String) null);
    }
 
    void require(ElementType type, String namespace, String name);
 
+   default void require(ElementType type, String namespace, Enum<?> name) {
+      require(type, namespace, name.toString());
+   }
+
+   Map.Entry<String, String> getMapItem(String nameAttribute);
+
+   Map.Entry<String, String> getMapItem(Enum<?> nameAttribute);
```

Comment:
I guess it could have a default implemention

Reply:
I actually have all of those in the `AbstractConfigurationReader` class